src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -146,7 +147,8 @@ public void printPdf(Claim claim, List<Printable> documents) {
             return document.template.getBytes();
         }
 
-        return pdfServiceClient.generateFromHtml(document.template.getBytes(), document.values);
+        byte[] html = pdfServiceClient.generateFromHtml(document.template.getBytes(), document.values);
+        return Base64.getEncoder().encodeToString(html).getBytes();
```

Comment:
```suggestion
        return Base64.getEncoder().encode(html);
```

No need to encode to `String` and back to `byte[]` again. Also, `getBytes` uses the platform default character encoding but `encodeToString` always uses ISO-8859-1 so you may run into encoding issues.

Reply:
yes it is failing , so looking into it